contracts/lib/PILFlavorsV2.sol

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.23;
+
+import { IPILicenseTemplate, PILTerms } from "../interfaces/modules/licensing/IPILicenseTemplate.sol";
```

Comment:
shouldn't this file be in periphery contract?

Reply:
yes, we could move PILicenseTemplate into the periphery repo.